File: leet/number_of_1_bits 191/number_of_1_bits.dart
/// done it using bit wise operation
class Solution {
  int hammingWeight(int n) {
    int count = 0;
    while (n != 0) {
      if ((n & 1) != 0) {
        count++;
      }
      n = n >> 1;
    }
    return count;
  }
}

File: leet/number_of_1_bits 191/number_of_1_bits.js
/**
 * @param {number} n
 * @return {number}
 */
var hammingWeight = function (n) {
    var count = 0;
    while (n !== 0) {
        if (n & 1 == 1) {
            count++;
        }
        n = n >> 1;
    }
    return count;
};